standards.md

```diff
@@ -195,23 +211,30 @@ what we want.
   * Error strings are not capitalized
   * Handle all errors ([gracefully](https://dave.cheney.net/2016/04/27/dont-just-check-errors-handle-them-gracefully))
     * When returning errors, add more context with `fmt.Errorf` and `%v`
-  * Use meaningful package names (avoid util, helper, lib)
   * Prefer short variable names
 
+### Go packages
+
+* Prefer small well factored packages with unit tests
+* Use meaningful package names (avoid util, helper, lib). See https://go.dev/blog/package-names
+* Only export functions that provide a meaningful API for consumers of the package
+* Test exported functions only, since these are the functions package consumers will use
```

Comment:
If we want to move this point under `Go package`, can we rephrase it to make it more focused on how we want to organize "Go packages" instead of how to write tests for go packages?

For example, instead of saying Test exported functionas only, we may want to rephrase it as something like "Every exported functions should be tested"? wdyt?

Reply:
thanks! reworded a bit